ec/ecdatasource/deploymentsdatasource/expanders.go

```diff
@@ -153,3 +169,32 @@ func newNestedTermQuery(path, term string, value interface{}) *models.QueryConta
 		},
 	}
 }
+
+// newNestedTagQuery returns a nested query for a metadata tag
+func newNestedTagQuery(path string, key interface{}, value interface{}) *models.QueryContainer {
```

Comment:
Won't the path always be `metadata.tags"`? I'm asking since it seems redundant to pass the `path` here.

Reply:
Yes, for this specific "tag" nested query the path will always be "metadata.tags" (hopefully !).
I've removed passing the value to the function in https://github.com/elastic/terraform-provider-ec/pull/248/commits/9b5e41744342eab132f239fce5675f6d3c5ada53.